self-monitor/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:alpine3.18 as builder
+RUN apk add --no-cache git make musl-dev go curl bash
+
+# Configure Go
+ENV GOROOT /usr/lib/go
+ENV GOPATH /go
+ENV PATH /go/bin:$PATH
+
+RUN mkdir -p ${GOPATH}/src ${GOPATH}/bin
+WORKDIR /
+RUN curl -LO https://github.com/prometheus/prometheus/archive/refs/tags/v2.50.1.tar.gz  && \
```

Comment:
Please use the LTS versions: https://prometheus.io/docs/introduction/release-cycle/

Reply:
done